README.md

```diff
@@ -17,6 +17,9 @@ PREFIX = a!
 SAUCENAO_API_KEY = <>
 WOLFRAM_API_KEY = <>
 DEEPL_API_KEY = <>
+CLIENT_ID = <>
+CLIENT_SECRET = <>
+REFRESH_TOKEN = <>
```

Comment:
Probably should rename these to something like `REDDIT_CLIENT_ID` (etc). That way you know what is needed here